ibm/service/kubernetes/resource_ibm_container_cluster.go

```diff
@@ -1273,6 +1305,19 @@ func resourceIBMContainerClusterUpdate(d *schema.ResourceData, meta interface{})
 
 	}
 
+	if d.HasChange("image_security_enforcement") {
```

Comment:
In create Method we are enabling image security...and create is calling the Update .
I think this condiiton gets satisfied and tries to enable the image security API call again...I think we need to add condiiton
d.HasChange("image_security_enforcement") && !d.IsNewResource() 

Same for VPC cluster resource also

Reply:
If the parameter is set to false from true after a while, would not having this in the code mean that an update would not unset image security?